23-freight-car-door-motion-and-location/README.md

```diff
@@ -0,0 +1,159 @@
+  - [Notehub Setup](#notehub-setup)
+  - [Hardware Setup](#hardware-setup)
+  - [Notecard Firmware](#notecard-firmware)
+  - [Notecard Configuration](#notecard-configuration)
+    - [`card.voltage`](#cardvoltage)
+    - [`card.location.mode`](#cardlocationmode)
+    - [`card.location.track`](#cardlocationtrack)
+    - [`card.aux` GPIO mode](#cardaux-gpio-mode)
+  - [Detecting when the door is opened and closed](#detecting-when-the-door-is-opened-and-closed)
+    - [Hall Effect Sensor](#hall-effect-sensor)
+    - [Magnetic Door Switch](#magnetic-door-switch)
+  - [Notecard CLI](#notecard-cli)
+
+
+## You will need
+
+* USB A to micro USB cable
+* [LiPo battery with JST cable](https://shop.blues.io/products/5-000-mah-lipo-battery)
+* [Notecarrier-B](https://shop.blues.io/products/carr-b)
+* [Notecard WBNA](https://shop.blues.io/collections/notecard/products/note-wbna-500)
+* To sense the door opening, either
+  * [Hall Effect Sensor](https://www.amazon.com/HiLetgo-NJK-5002C-Proximity-3-Wires-Normally/dp/B01MZYYCLH/)
+* or
+  * [Magnetic Door Sensor](https://www.amazon.com/Gufastore-Surface-Magnetic-Security-Adhesive/dp/B07F314V3Z)
+* Dupont jumper cables (male-female)
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub).
+
+## Hardware Setup
+
+1. Assemble Notecard and Notecarrier as described [here](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-b).
+2. Plug the LiPo battery's JST cable into the Notecarrier port labeled "LIPO".
+3. Connect the micro USB cable from your development PC to the port on the Notecarrier. This is to both charge the LiPo battery and configure the Notecard.
+
+## Notecard Firmware
+
+The Notecard should use [firmware version 3.5.2](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-2-november-2nd-2022) or higher. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
+
+
+## Notecard Configuration
+
+With the Notecarrier connected to your development PC, navigate to https://dev.blues.io, open the in-browser terminal and connect to the Notecard. Then enter this command in the terminal:
+
+```json
+{ "req": "hub.set", "product" : "com.your-company:your-product-name", "sn": "a-serial-number" }
+```
+
+Make sure to replace "com.your-company:your-product-name" with your ProductUID from [Notehub Setup](#notehub-setup). Also, replace "a-serial-number" with the serial number of other relevant identifier for this particular tracker (e.g. "freight car 2").
+
+### [`card.voltage`]()
+
+Next, enter this command to optimize voltage ranges for the LiPo battery
+
+```json
+{ "req": "card.voltage", "mode": "lipo" }
+```
+
+
+And then this command
+
+```json
+{ "req": "hub.set", "mode": "periodic", "inbound": 1440, "outbound": 60 }
```

Comment:
Not strictly necessary, but should we merge the extra parameters here with the first hub.set command?

Reply:
I wanted to keep them separate so that the project/sn is distinct from the connectivity parameters. Not a strong opinion, just thought it was easier to document like that.